server/app.js

```diff
@@ -4,6 +4,8 @@ const bodyParser = require('body-parser');
 
 const path = require('path');
 
+const Reviews = require('../database/index.js')
```

Comment:
So you require the db index file, which initializes a connection to mongoose. I'm fairly confident this is why your circleci build is failing, because this file gets required by the tests.

I'm pretty sure you can fix this by putting your schema into a separate file from the DB index where you connect to mongoose. You don't actually need to connect to the db to define/use a schema. Also you don't need to connect to the db in this file, you can do it in the server index